lib/AST/ASTPrinter.cpp

```diff
@@ -1865,22 +1881,6 @@ void PrintAST::printOneParameter(const ParamDecl *param, bool Curried,
     }
   }
```

Comment:
This one is different from the others. Skipping either branch causes a bunch of tests to fail.